packages/uniforms-material/src/SelectField.tsx

```diff
@@ -162,7 +150,8 @@ function Select(props: SelectFieldProps) {
 
   return (
     <TextField
-      disabled={!!disabled}
+      {...filterDOMProps(props)}
```

Comment:
It looks like we cannot do that - there's `...filterDOMProps(props)` in `SelectProps` in line 172.